pkg/kubectl/cmd/util/cmdtools.go

```diff
@@ -170,23 +170,31 @@ type TempFile struct {
 	Buffer io.Reader
 }
 
-// LaunchTempFile reads the provided stream into multiple temporary file in the given directory
+// LaunchTempFiles reads the provided stream into multiple temporary file in the given directory
 // and file prefix, and then invokes Launch with the path of that file. It will return
 // the contents of the first file after launch, any errors that occur, and the path of the
 // temporary file so the caller can clean it up as needed.
-func (t CmdTool) LaunchTempFile(tmpFile ...TempFile) ([]byte, string, error) {
+func (t CmdTool) LaunchTempFiles(tmpFile ...TempFile) ([]string, error) {
 	var paths []string
 	for _, f := range tmpFile {
 		path, err := prepareTempFile(f.Prefix, f.Suffix, f.Buffer)
 		if err != nil {
-			return nil, path, err
+			return []string{""}, err
```

Comment:
I just want to consist with return value of `prepareTempFile`, but I think returnning `nil` is also fine, I will update it to return `nil`